web/src/views/InvestigationDetail.vue

```diff
@@ -151,17 +172,82 @@ export default defineComponent({
       geojsParams.layer.crossDomain = 'use-credentials';
       createMap(geojsParams.map);
       createLayer('osm', geojsParams.layer);
+      clampBoundsX(false);
+    });
+    // #endregion datasets
+
+    // #region pin
```

Comment:
Is this like a folding support for an IDE or something?

Reply:
Yeah, it is. I was not thinking of the fact that there's no standard tool people use. I don't mind removing the region markers if that's best practice.

Reply:
It's probably best to remove them.

What IDE are you using? I know vim does this sort of thing in-band but in modern editors you ought to be able to do it out-of-band (so to speak).

Reply:
vscode

Reply:
Removed in e8f054f